packages/utils/package.json

```diff
@@ -24,6 +24,11 @@
       "require": "./dist/cli-utils.js",
       "import": "./dist/cli-utils.mjs",
       "types": "./dist/cli-utils.d.ts"
+    },
+    "./assets-utils": {
+      "require": "./dist/assets-utils.js",
+      "import": "./dist/assets-utils.mjs",
+      "types": "./dist/assets-utils.d.ts"
```

Comment:
I wonder if it should be inside the new `account` package, following the `Provider` route:
 - #1675

> _cc @Torres-ssf_

Reply:
@arboleya Makes sense. Do you believe I should wait for that PR to be merged, or can I work on this in parallel?

Reply:
There are things you could do ahead of time, or you could also branch off of that branch/PR.

That's entirely up to you.

Reply:
@arboleya I agree, this can be part of the `account` package.

Reply:
Maybe it could even go inside `providers`:
 - `accounts/src/providers/chains.ts`
 - `accounts/src/providers/assets/index.ts`
 - `accounts/src/providers/assets/icons/..`

Reply:
why inside `providers` ? assets has nothing to do with network/account... they can be even from other chain

I may lack some context here but `@fuel-ts/assets` feels good to me

Reply:
I agree; however, over-categorizing small chunks of code led us to have dozens of packages flattened out in the repo root without an apparent relationship between them, which created a maintenance burden we are trying to reduce by re-grouping things in [this PR](https://github.com/FuelLabs/fuels-ts/pull/1675) as an attempt to defrag the list of packages. Things might not look perfect and are subject to change, but we must start somewhere, and flexibility is required to get things moving.

Nevertheless, everything will still be available from `fuels`.